### DISCUSSION TITLE
Technical Problem

### DISCUSSION THREAD

**Discussion by syndicate500** [winstxnhdw/lc-hax#155] [Q&A] [answered]
already read README and follow the instructions, but show this error
![image](https://github.com/winstxnhdw/lc-hax/assets/29987349/29efe37c-01cc-458d-8469-f22980bc44ea)

**Comment by winstxnhdw** [comment, marked as answer]
There's no error. It's working hahaha.

**Reply by syndicate500** [reply]
@winstxnhdw working, but i cant still use the command in game, i tried typing any command ingame, but do nothing

**Reply by winstxnhdw** [reply]
Are you typing in chat?

**Reply by winstxnhdw** [reply]
Seems like you solved it.
